### ISSUE TITLE
Fail to run test_load_store with default settings

### ISSUE THREAD

**Issue by pengfeix** [harvard-acc/gem5-aladdin#35]
**I ran the default run.sh script for test_load_store example and got the following output:**
`warn: DRAM device capacity (8192 Mbytes) does not match the address range assigned (4096 Mbytes)
0: system.remote_gdb: listening for remote gdb on port 7000
info: Entering event queue @ 0.  Starting simulation...
info: Increasing stack size by one page.
warn: ignoring syscall access(...)
warn: x86 cpuid family 0x0000: unimplemented function 2
warn: x86 cpuid family 0x0000: unimplemented function 2
warn: x86 cpuid family 0x0000: unimplemented function 2
warn: x86 cpuid family 0x0000: unimplemented function 2
info: Received mapping for array store_vals at vaddr 6d0810 of length 8192.
info: Received mapping for array store_loc at vaddr 6d2820 of length 8192.
[WARNING]: Overlapping array declarations found!
  store_vals: 0x226e7d0 - 0x226e7d0
  store_loc: 0x22707e0 - 0x22727e0
[WARNING]: Overlapping array declarations found!
  store_loc: 0x22707e0 - 0x22727e0
  store_vals: 0x226e7d0 - 0x226e7d0
Overlapping array address ranges can lead to incorrect behavior, such as DMA nodes trying to access the wrong arrays, ACP accessing the wrong memory, etc. Please check your Aladdin configuration file and the mapArrayToAccelerator() calls to verify that they are correct.
Global loop pipelining is not ON.
 Current power model supports trig functions running at 10 ns. 
 Cycle time: 2 is not supported yet. Use 10ns power model instead.
`
**And in stdout.gz, I got the output:**
`===============================
        Aladdin Results
===============================
44938000: system.test_load_store_datapath: Accelerator completed.
44938000: system.test_load_store_datapath: Elapsed host seconds 0.39.
44938000: system.test_load_store_datapath: Sent finished signal.
44954000: system.test_load_store_datapath: cacheRespCallback for control signal access: 0xd9860
44956000: system.test_load_store_datapath: Woken up the CPU thread context.
Accelerator finished!
FAILED: store_loc[0] = -1, should be 0
FAILED: store_loc[1] = -1, should be 1
FAILED: store_loc[2] = -1, should be 2
FAILED: store_loc[3] = -1, should be 3
FAILED: store_loc[4] = -1, should be 4
FAILED: store_loc[5] = -1, should be 5
...
...
FAILED: store_loc[2045] = -1, should be 2045
FAILED: store_loc[2046] = -1, should be 2046
FAILED: store_loc[2047] = -1, should be 2047
Test failed with 2048 errors.Exiting @ tick 3275724000 because exiting with last active thread context
Simulated exit code not 0! Exit code is 255
`

Thanks.

**Comment by xyzsam** [comment]
You can ignore the overlapping array warnings. Are you running this inside Docker?

**Comment by pengfeix** [comment]
No, I compile and run this on Ubuntu16.04. I think the error is that I don't get "Test passed!" in stdout.gz file. I wil try to run this in docker.
